src/core/particles/ParticleContainer.js

```diff
@@ -33,10 +33,25 @@ var Container = require('../display/Container'),
  * @param [properties.uvs=false] {boolean} When true, uvs be uploaded and applied.
  * @param [properties.alpha=false] {boolean} When true, alpha be uploaded and applied.
  */
-function ParticleContainer(size, properties)
+function ParticleContainer(totalSize, properties, batchSize)
 {
     Container.call(this);
 
+    batchSize = batchSize || 15000; //CONST.SPRITE_BATCH_SIZE; // 2000 is a nice balance between mobile / desktop
```

Comment:
Why did the default change to 15k? The problem this is trying to solve is not updating a large buffer that is mostly unused, but this actually will make that worse for most people by default.